src/editor/Editor.js

```diff
@@ -99,7 +99,6 @@ define(function (require, exports, module) {
         USE_TAB_CHAR        = "useTabChar",
         WORD_WRAP           = "wordWrap",
         AUTO_HIDE_SEARCH    = "autoHideSearch",
-        INDENT_LINE_COMMENT   = "indentLineComment",
```

Comment:
This file should not be present in the commit. Exclude this file. Don't delete it in PR.